roles/http-agent/templates/http_agent_service.yaml.j2

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    http-agent: k8s
```

Comment:
This is commented on `http_agent_external_service.yaml.j2`.
If it will stay commented in the other file, you should comment here too.